test/TestRist.cpp

```diff
@@ -5,8 +5,107 @@
 
 #include "RISTNet.h"
 
-std::string kValidPsk = "Th1$_is_4n_0pt10N4L_P$k";
-std::string kInvalidPsk = "Th1$_is_4_F4k3_P$k";
+const std::string kValidPsk = "Th1$_is_4n_0pt10N4L_P$k";
+const std::string kInvalidPsk = "Th1$_is_4_F4k3_P$k";
+
+const uint16_t kConnectTimeout = 2;
```

Comment:
Use std::chrono::seconds already here to clearly state the unit :)

Reply:
Ah, of course!